bin/v-setup-server-smtp-account

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
We should move it to /install/update/manual/  and replace it this script so we can automate it later on via CLI / php